test/runtime/gbt/comm/bigBulkTran.compopts

```diff
@@ -0,0 +1 @@
+--fast
```

Comment:
Why `--fast`? We're trying to avoid throwing --fast in compopts if possible. If there's some value in it I'm not opposed. If it's because bounds-checking slows this test down you could either just throw `--no-bounds-checks` or you could rewrite your initializer loops to be:

```chpl
forall (a,  i) in zip(A, A.domain) do a = i;
```

Reply:
Yes, it's just for execution time reduction.  I'll switch to `--no-bounds-checks`; it seems to work nearly as well.